.github/workflows/changesets.yml

```diff
@@ -4,7 +4,6 @@ on:
   push:
     branches:
       - master
-  pull_request:
```

Comment:
yes the error was annoying. i'll check this later - to see if we need to use `pull_request_target` or just remove it

Reply:
Since it generates a PR based on the change set committed, I suppose it's only valid on pushed to master - it's not something to verify for a PR.

Reply:
it actually for the sanity-check since we'll run some scripts like `prepack` and `prepublishOnly` before creating the PR (e.g. a script that changes alteration scripts from `next-*` to `v1.0-*`). but you are right, we and fix it later as long as it runs on pushes.